src/EventStore.Core.Tests/ClientOperations/specification_with_request_manager_integration.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using EventStore.Core.Bus;
+using EventStore.Core.Messages;
+using EventStore.Core.Messaging;
+using EventStore.Core.Tests.Services.Replication;
+using NUnit.Framework;
+
+namespace EventStore.Core.Tests.ClientOperations {
+	public abstract class specification_with_request_manager_integration : specification_with_bare_vnode {
+		
+		protected long CompletionMesageCount;
+		protected StorageMessage.RequestCompleted CompletionMessage;
+
+		protected Guid InternalCorrId = Guid.NewGuid();
+		protected Guid ClientCorrId = Guid.NewGuid();
+		protected byte[] Metadata = new byte[255];
```

Comment:
We can safely remove `Metadata` and `EventData` as these fields are not being used.

Reply:
fixed